guardian/manage-frontend
# Lc css overides

## What does this change?
Additional css styling for the live chat window and its contents where the salesforce lightning web components don't offer enough control.

## How to test
- in helpCentre.tsx make sure the `isLiveChatReleased` variable is set to true
- log into the dev salesforce envirmonment
- from the App Launcher menu select LiveChatTestApp
- in the omni channel window set your self as Available - chat
- run manage locally and visit the help center with the live chat url param: `https://manage.thegulocal.com/help-centre/?liveChat=1` 

## Images

### live chat window on mobile breakpoint
![Screenshot 2021-08-19 at 10 15 54](https://user-images.githubusercontent.com/2510683/130043465-0cbe961e-f248-4e2b-be2a-042d73bf5131.png)

### live chat window on desktop breakpoint
![Screenshot 2021-08-19 at 10 16 14](https://user-images.githubusercontent.com/2510683/130043513-f2552e5e-56ab-4bfa-b3cc-1b1e6966dcc3.png)

### waiting to connect state screen
![Screenshot 2021-08-19 at 10 16 39](https://user-images.githubusercontent.com/2510683/130043592-1ff0e073-fed9-4add-9c54-a4f87c1dd182.png)

### leave in waiting state
![Screenshot 2021-08-19 at 10 16 57](https://user-images.githubusercontent.com/2510683/130043683-18a3cc01-b3d3-4002-a093-c20e8f33935b.png)

### main chat window
![Screenshot 2021-08-19 at 10 18 00](https://user-images.githubusercontent.com/2510683/130043781-a3e619ab-cbde-4f42-8b56-99e5b7893a05.png)

### user ends chat
![Screenshot 2021-08-19 at 10 16 57](https://user-images.githubusercontent.com/2510683/130043855-976e3ed7-1581-46df-a611-b03e0c8219d2.png)

### CSR ends chat
![Screenshot 2021-08-19 at 10 18 20](https://user-images.githubusercontent.com/2510683/130043902-e3ba33ab-367f-4dcf-9c1a-bbb21ed12b5c.png)
